src/System.Data.Odbc/tests/Configurations.props

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <BuildConfigurations>
```

Comment:
You are going to need configurations similar to these. What issues where you hitting that caused you to remove them?